stdlib/2and3/ctypes/__init__.pyi

```diff
@@ -0,0 +1,257 @@
+# Stubs for ctypes
+
+from typing import (
+    Any, Callable, ClassVar, Iterable, List, Mapping, Optional, Sequence, Sized, Tuple, Type,
+    Generic, TypeVar, overload,
+)
+from typing import Union as UnionT
+import sys
+
+_T = TypeVar('_T')
+if sys.platform == 'win32':
+    _DLLT = TypeVar('_DLLT', CDLL, OleDLL, WinDLL, PyDLL)
+else:
+    _DLLT = TypeVar('_DLLT', CDLL, PyDLL)
+_CT = TypeVar('_CT', bound=_CData)
+
+
+RTLD_GLOBAL: int = ...
+RTLD_LOCAL: int = ...
+DEFAULT_MODE: int = ...
+
+
+class _DLL:
+    def __init__(self, name: str, mode: int = ..., handle: Optional[int] = ...,
+                 use_errno: bool = ..., use_last_error: bool = ...) -> None: ...
+    def __getattr__(self, name: str) -> Any: ...
+class CDLL(_DLL): ...
+if sys.platform == 'win32':
+    class OleDLL(_DLL): ...
+    class WinDLL(_DLL): ...
+class PyDLL(_DLL):
+    _handle: int = ...
+    _name: str = ...
+    def __init__(self, name: str, mode: int = ...,
```

Comment:
Isn't the `__init__` just inherited from the base class? https://github.com/python/cpython/blob/2.7/Lib/ctypes/__init__.py#L389

Reply:
Technically yes, but [the documentation](https://docs.python.org/2.7/library/ctypes.html#ctypes.PyDLL) lists the constructor as not accepting the `use_errno` and `use_last_error` kwargs. I don't know what the correct procedure is in cases like this - is the documentation or the code right?

Reply:
I'd generally favor the code.

Reply:
OK, I've removed the `__init__` from `PyDLL`, it should now inherit the one from `CDLL` again.